packages/opentelemetry-instrumentation-http/test/functionals/http-disable.test.ts

```diff
@@ -14,29 +14,33 @@
  * limitations under the License.
  */
 import { NoopTracerProvider, NOOP_TRACER } from '@opentelemetry/api';
-import { NoopLogger } from '@opentelemetry/core';
+import { NoopLogger, isWrapped } from '@opentelemetry/core';
```

Comment:
isWrapped is part of instrumentation

Reply:
We should remove it from the core then (in another PR though)